comfy_cli/command/distribution.py

```
@@ -1184,6 +1523,100 @@ def update_cmd(
     renderer.emit(dist, command="distribution update", changed=True)
 
 
+@app.command("from-snapshot", help="Create a distribution from a ComfyUI Desktop snapshot export.")
+@tracking.track_command("distribution")
+def from_snapshot_cmd(
```

Pylint (4.0.6)

[convention] Missing function or method docstring

(C0116)